xml/depl_maintenance.xml

```diff
@@ -1625,8 +1627,8 @@ migrate</screen>
        <itemizedlist>
 	<listitem>
 	 <para>
-	  <filename>/var/log/nova/nova-compute</filename> on the source and
-	  target &compnode;s.
+	  <filename>/var/log/nova/nova-compute</filename> on the &compnode;s
+	  that the migration is performed from and to.
```

Comment:
"... on the source and origin &compnode;s."

Reply:
_the &compnode;s that the migration is performed from and to_ is admittedly a bit tortured, but it conveys the intended meaning better, methinks.